WordPress/Classes/ViewRelated/Aztec/ViewControllers/AztecPostViewController.swift

```diff
@@ -3498,8 +3485,6 @@ extension AztecPostViewController {
             return
```

Comment:
The reason why this code is here, is no longer intuitive IMHO: https://github.com/wordpress-mobile/WordPress-iOS/pull/13134/files#diff-705f087506da53949ae67cbac0ed8d98L3497-L3499

Two questions:
1. Should this be removed?
2. If not - any chance we can add a note as to why this would make the editor bail out?

Reply:
> Should this be removed?

If no actions were made in the image we do not replace it. Otherwise, we will duplicate and re-upload the image. More about this [here](https://github.com/wordpress-mobile/WordPress-iOS/commit/6f1b0f666c7848df6cf7d62e0be13d3b68b5e641).

> If not - any chance we can add a note as to why this would make the image replacement bail out?

Sure!

Reply:
Done in 9f502ffdaee34baf4bd4da70cb9ea6034fd1055d